packages/inngest/src/hono.ts

```diff
@@ -35,11 +36,52 @@ export const serve = (options: ServeHandlerOptions) => {
         transformResponse: ({ headers, status, body }) => {
           return c.body(body, { headers, status });
         },
-        url: () => new URL(c.req.url, c.req.header("host")),
+        url: () => {
```

Comment:
Would `canParse` help here? it's safe to use becasue anyone using Hono would be using a runtime that supports it. https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static

Reply:
Ah, awesome. Aye, at a glance it looks like Hono supports 18+. Thanks!